### PR TITLE
EBP-910 add user id query to fetch user id

### PR DESCRIPTION
**Pull request by sourabhmandal**

### CODE REVIEW

File: jwt.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+func (m *middlewareStruct) GetUserID(baseUrl string) (GetUserIDResponse){
+	api := baseUrl + "/getUserPublicProfile"
+	response, err := http.Get(api)
+	if err != nil {
+		fmt.Print(err.Error())
```
Comment (srivallabh-mangrulkar) [inline jwt.go]: Use log instead of fmt

**Review by srivallabh-mangrulkar — CHANGES_REQUESTED** [review]
We are using the verifyJwtToken method in every service, what is the rationale behind removing it?

**Review by ravizbyte — APPROVED** [review]

File: jwt.go (hunk 1 above)
Comment (sourabhmandal) [reply jwt.go]: okay

**Comment by sourabhmandal** [comment]
> We are using the verifyJwtToken method in every service, what is the rationale behind removing it?

According to the discussion, all token verification issuing and refreshing will be done by the new auth service.
This middleware layer will later become the gateway layer and the middleware-lib will retire.

- To avoid rework later, less logic is implemented in the middleware layer.
- Push toward single responsibility, auth service should handle all JWT stuff

**Review by ghost — APPROVED** [review]

**Review by ravizbyte — APPROVED** [review]

**Review by srivallabh-mangrulkar — APPROVED** [review]

**Pull request merged**